src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2193,21 +2193,10 @@ internal bool IsNonMoveableVariable(BoundExpression expr, out Symbol accessedLoc
                         }
                     case BoundKind.PointerIndirectionOperator: //Covers ->, since the receiver will be one of these.
                     case BoundKind.PointerElementAccess:
-                    case BoundKind.StackAllocArrayCreation:
+                    case BoundKind.ConvertedStackAllocExpression:
```

Comment:
Can this method be called with BoundStackAllocArrayCreation anymore?

Reply:
No. This API is for internal testing only on valid compilations.

---
In reply to: [136683286](https://github.com/dotnet/roslyn/pull/21746#discussion_r136683286) [](ancestors = 136683286)